app/controllers/idv/enter_password_controller.rb

```diff
@@ -163,6 +163,10 @@ def password
     end
 
     def confirm_no_profile_yet
+      if current_user.profiles.where('activated_at is NOT NULL').any?
```

Comment:
While this works could we possibly make this a method in user.rb called #has_proofed_before? so we can remember what this is checking for?

Reply:
yes absolutely! added in https://github.com/18F/identity-idp/pull/12103/commits/ea24ce49d20de6972ebefcff8b864084aa98a912 and this way i can test it more explicitly, too.